ethcore/sync/src/chain/mod.rs

```diff
@@ -1183,7 +1183,7 @@ pub mod tests {
 	}
 
 	pub fn get_dummy_blocks(order: u32, parent_hash: H256) -> Bytes {
-		let mut rlp = RlpStream::new_list(1);
+		let mut rlp = RlpStream::new_list(2);
```

Comment:
So this was it? Our test was wrong the whole time? :)

Reply:
yes :(